specs/data_structures.md

```diff
@@ -330,7 +330,7 @@ For leaf node `node` of [share](#share) data `d`:
 ```C++
 node.n_min = d.namespaceID
 node.n_max = d.namespaceID
-node.v = h(0x00, d.rawData)
+node.v = h(0x00, d.namespaceID, d.rawData)
```

Comment:
I don't know how often we've already talked about this line :-D

Reply:
Yeah I'm sure we even discussed adding in the namespace ID here, but not in the context of the recovering case.